expressEngine/customer/service/customer.js

```diff
@@ -46,6 +47,22 @@ class Service {
         ? { message: success.customerDeleted }
         : { error: error.customerNotFound };
     };
+
+    put = async (body) => {
+      const { id, dataToUpdate } = body;
+      if (!dataToUpdate || Object.entries(dataToUpdate).length === 0) {
+        return { error: error.emptyData };
+      }
+      if (dataToUpdate.email && !modelLib.validateEmail(dataToUpdate.email)) {
+        return { error: error.notUpdated };
+      }
+      const result = await DBService.updateOne(Customer, { _id: id }, dataToUpdate);
+      if (result.error) {
+        return result;
+      }
+      const { nModified } = result;
+      return nModified ? { message: success.customerUpdated } : { error: error.notUpdated };
```

Comment:
@rahuljainss please return alreadyUpdated error also instead of notUpdated.

Reply:
@chiragSuccessive I think not Updated error is better than Already Updated, If User updated value and the previous value is the same, there is no update operation involved, so already updated will be a wrong message